markdown/stencil-docs/developing-further/theme-updates-and-version-control.md

```diff
@@ -1,161 +1,46 @@
-
-### Downloading the Cornerstone Repository
-
-This approach downloads all theme files at once, allowing you to copy/paste their relevant contents locally.
-
-1. Navigate to the [Cornerstone Github Repository](https://github.com/bigcommerce/cornerstone).
-2. On the right hand side of the page, click the green `Clone or download` button, and select Download ZIP.
-
-
-
-
-
-<a href='#theme-updates_synchronizing-multiple' aria-hidden='true' class='block-anchor'  id='theme-updates_synchronizing-multiple'><i aria-hidden='true' class='linkify icon'></i></a>
-
-## Synchronizing Updates for Multiple Themes
-
-These tips run from simpler to more-complex scenarios, with a similar progression from simpler to more-complex tools.
-
-### Enlist Your Clients in Future-Proofing
-Warn your clients that editing their themes' .html files (via Edit Theme Files) will complicate future theme upgrades. Ask clients to keep a record of such changes, so that when you release a theme update, they can reapply their changes themselves.
-
-### Max Out Variations
-For very simple theme differences (like changing color values), you can maintain a different theme variation per storefront or use case. Given Stencil's limit of four variations per theme, this approach obviously has limited scalability.
+---
 
-### Customize Outside the Theme
-Rely as much as possible on dynamic aspects of the BigCommerce control panel, like the Footer Scripts editor. (Control‑panel customizations are saved per store, without complicating your theme's codebase.) For other examples of control-panel options, please see Dynamic Content Injection above.
+<a id="markdown-maintain-customizations" name="maintain-customizations"></a>
 
-### Use Conditional Logic within Master Templates
-For subtle changes to a portion of a template file, use Handlebars helpers and conditionals to render different HTML for each client's flavor of the theme.
+## Maintaining Customizations
 
-### Parallel GitHub Repo's
-Where different clients' theme flavors diverge into completely custom pages – for example, each client has a wildly different homepage layout – Git's systematic version control helps. You will probably want to fork or branch separate repo's, one per client, and maintain/update them in parallel.
+A way to keep your customizations separate from the base theme is to rename the files. For example, `{{{stylesheet ‘/assets/css/theme.css’}}}`is the theme CSS that comes with the file. Instead of making edits to this file create a new one `{{{stylesheet ‘/assets/css/my_custom_theme.css’}}}` and reference this file in the layouts.
 
-As with relying on theme variations, this approach has limited scalability. Beyond a certain number of clients/themes (certainly by 10), it becomes cumbersome.
+Make sure to discuss with the merchant not to make any changes to the files. This can cause issues later on with updates. 

```

Comment:
It may be helpful to discuss with the merchant not to make any changes to the files. Having multiple editors to files could potentially cause issues later on with updates if a proper workflow is not established.

Reply:
This illustrates just how different we write. Remember when Karen was like Tatiana you are not very helpful and Nikita you might be too helpful. 🤣 💙